server/src/server.ts

```diff
@@ -91,7 +99,39 @@ connection.onCompletion((textDocumentPosition: TextDocumentPositionParams): Comp
   if (documentAsText === undefined) {
     return []
   }
-  return contextHandler.getComletionItems(textDocumentPosition, documentAsText)
+
+  const word = analyzer.wordAtPointFromTextPosition({
+    ...textDocumentPosition,

```

Comment:
It seems the name of this variable is misleading. Maybe it should be renamed "textDocumentPositionParams" or just be destructured from the beginning?

Reply:
yep, it should be params. The original name was there at the very beginning.